boards/arm/nucleo_l433rc_p/arduino_r3_connector.dtsi

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpioa 0 0>,	/* A0 */
+			   <1 0 &gpioa 1 0>,	/* A1 */
+			   <2 0 &gpioc 3 0>,	/* A2 */
+			   <3 0 &gpioc 2 0>,	/* A3 */
+			   <4 0 &gpioc 1 0>,	/* A4 */
+			   <5 0 &gpioc 0 0>,	/* A5 */
+			   <6 0 &gpioa 3 0>,	/* D0 */
```

Comment:
Arduino D0 is connected to PA10
D1 to PA9

Reply:
On the quick reference card it shows D0 PA3/PA10, D1 PA2/PA9. On the schematic PA2, PA3 connections can be made by jumpers. I'll change this to the default (PA9, PA10) as suggested by you.